go/mysql/server.go

```diff
@@ -343,6 +343,11 @@ func (l *Listener) handle(conn net.Conn, connectionID uint32, acceptTime time.Ti
 				return
 			}
 		case ComQuery:
+			// flush is called at the end of this block.
```

Comment:
Does this mean that `*bufio.Writer` won't be returned to pool in case of error? What are the problems with returning it?

Reply:
Good catch. I need to think about this...